ElectronicObserver/Window/Wpf/Fleet/ViewModels/FleetItemViewModel.cs

```diff
@@ -612,4 +627,58 @@ private string GetEquipmentString(IShipData ship)
 		return sb.ToString();
 	}
 
+	private void AddDaySpecialAttacksToTooltip(IShipData ship, StringBuilder sb, EngagementType engagement, IFleetData fleet)
+	{
+		if (!SpecialAttackHitList.Any(specialAttack => specialAttack.Key.CanTriggerOnDay)) return;
+
+		sb.Append($"\r\n{FormFleet.SpecialAttacksDay}");
+
+		foreach ((SpecialAttack attack, List<SpecialAttackHit> hits) in SpecialAttackHitList.Where(specialAttack => specialAttack.Key.CanTriggerOnDay))
+		{
+			foreach (SpecialAttackHit hit in hits.Distinct())
+			{
+				double power = ship.GetDayAttackPower(attack, hit, fleet, engagement);
+				double accuracy = ship.GetDayAttackAccuracy(hit, fleet);
+
+				AddSpecialAttackToTooltip(sb, attack, hit, power, accuracy);
+			}
+		}
+	}
+
+	private void AddNightSpecialAttacksToTooltip(IShipData ship, StringBuilder sb, EngagementType engagement, IFleetData fleet)
+	{
+		if (!SpecialAttackHitList.Any(specialAttack => specialAttack.Key.CanTriggerOnNight)) return;
+
+		sb.Append($"\r\n{FormFleet.SpecialAttacksNight}");
+
+		foreach ((SpecialAttack attack, List<SpecialAttackHit> hits) in SpecialAttackHitList.Where(specialAttack => specialAttack.Key.CanTriggerOnNight))
+		{
+			foreach (SpecialAttackHit hit in hits.Distinct())
+			{
+				double power = ship.GetNightAttackPower(attack, hit, fleet, engagement);
+				double accuracy = ship.GetNightAttackAccuracy(hit, fleet);
+
+				AddSpecialAttackToTooltip(sb, attack, hit, power, accuracy);
+			}
+		}
+	}
+
+	private void AddSpecialAttackToTooltip(StringBuilder sb, SpecialAttack attack, SpecialAttackHit hit, double power, double accuracy)
+	{
+		string attackDisplay = attack.GetDisplay();
+
+		if (hit.ShipIndex == 0)
+		{
+			sb.Append($"\r\n・[{AttackRateDisplay(attack.GetTriggerRate())}] - {attackDisplay}");
+		}
+		else
+		{
+			sb.Append($"・[{FormFleet.Helper}] - {attackDisplay}");
```

Comment:
I don't think the helper text is really needed, it should be good enough to just display the data.

Reply:
Helper text was just here in case we get special attacks that are not triggered by the flagship one day

Reply:
Well, the ship that can activate the attack would have the activation rate next to it. My line of thought is "there's no activation rate for this attack, so this ship can't activate it, but is just part of it".